mapping_node:
  ros__parameters:
    min_distance_to_match: 0.5
    num_iterations: 30
    use_reciprocal: true
    normal_search_radius: 0.1
    icp_method: p2point
    occupancy_grid_resolution: 0.025
